Cargo.toml

```diff
@@ -113,6 +132,13 @@ proptest = "1"
 rand_chacha = "0.3"
 rand_xorshift = "0.3"
 
+# Tor
+# - `arti-client` depends on `rusqlite`, and a version mismatch there causes a compilation
+#   failure due to incompatible `libsqlite3-sys` versions.
+arti-client = { version = "0.11", default-features = false, features = ["compression", "rustls", "tokio"] }
```

Comment:
After #1421 we are no longer constrained by `ring`, but we are still constrained here by `rusqlite` which is why I only bumped this from `0.9` to `0.11`, not to the latest `0.19`. We should separately look at bumping `rusqlite`, subject to the CocoaPods issue above.

Reply:
Ugh, we can't actually avoid cross-version `ring` dependencies, because `tor-rtcompat` depends on both a `rustls` version that uses `ring 0.17`, and the `x509-signature` crate that uses `ring 0.16`. We'd need to use the older `arti-client` version etc. that uses the older `rustls`. Given that the most recent release of Arti is also still in this state, I'm inclined to just leave it as-is (we're at least down to a single version of `rustls`) and see how it goes.

Reply:
I confirmed that this still happens to work on iOS, so I think we should just leave it as-is for now, and if I have time I'll try to contribute a change to Arti that migrates away from `x509-signature`.